browser/src/core/Socket.js

```diff
@@ -1649,8 +1649,8 @@ app.definitions.Socket = L.Class.extend({
 		this._map.fire('hyperlinkclicked', {url: link, coordinates: coords});
 	},
 
-	_onSocketError: function () {
-		window.app.console.debug('_onSocketError:');
+	_onSocketError: function (event) {
+		window.app.console.warning('_onSocketError:', event);
```

Comment:
shouldn't be: `window.app.console.warn` ?
example: https://github.com/CollaboraOnline/online/blob/3b7a26e15f866fa53fe3e49c01432b3d653a38f1/browser/src/core/Socket.js#L994

Reply:
Indeed. I always trip on this

https://developer.mozilla.org/en-US/docs/Web/API/console/warn_static